public/v4/apps/pigallery2.yml

```diff
@@ -0,0 +1,37 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        image: bpatrik/pigallery2:$$cap_version
+        environment:
+            NODE_ENV: production # set to 'debug' for full debug logging
+        volumes:
+            - $$cap_appname-config:/app/data/config
+            - $$cap_appname-db:/app/data/db
+            - $$cap_images:/app/data/images
+            - $$cap_appname-tmp:/app/data/tmp
+        restart: always
+volumes:
+    $$cap_appname-db:
+caproverOneClickApp:
+    variables:
+        - id: $$cap_version
+          label: PiGallery2 Version
+          defaultValue: '1.9.5'
+          description: Check out their docker page for the valid tags https://hub.docker.com/r/bpatrik/pigallery2/tags
+          validRegex: /^([^\s^\/])+$/
+        - id: $$cap_images
+          label: Set path to images
+          defaultValue: /home/User/MyPictures
```

Comment:
Let's remove this default value as people just click on deploy.
Also add a regex to ensure the value starts with a `/`